src/SFA.DAS.Commitments.Api/Orchestrators/EmployerOrchestrator.cs

```diff
@@ -215,5 +218,37 @@ public EmployerOrchestrator(IMediator mediator, ICommitmentsLogger logger)
                 UserId = userId
             });
         }
+
+        public async Task<GetPendingApprenticeshipUpdateResponse> GetPendingApprenticeshipUpdate(long accountId, long apprenticeshipId)
+        {
+            _logger.Info($"Getting pending update for apprenticeship {apprenticeshipId} for employer account {accountId}");
```

Comment:
I mean using the optional parameters at the end of the logger method call to log out the values for things like accountId, providerId, apprenticeshipId etc. This adds them as their own properties to elasticsearch.

eg. _logger.Info($"Getting commitment {commitmentId} for employer account {accountId}", **accountId: accountId, commitmentId: commitmentId**);